eng/scripts/docs/Docs-ToC.ps1

```diff
@@ -0,0 +1,23 @@
+function Get-NamepspacesFromDll($dllPath) {
```

Comment:
perhaps rename this to be Docs-ToC-Helpers.ps1 so it is a little clearer the purpose.

Reply:
The docs name aligned with what we have in Java/Python/JS.

Reply:
Ok.

Reply:
I prefer to stay as it is, so it is easily searchable.

Reply:
I (who came up with the `Docs-ToC.ps1` name) can see why this would be confusing. In the future we should have "Helper" or something in the name.